cluster-autoscaler/cloudprovider/oci/nodepools/oci_manager.go

```diff
@@ -82,7 +82,13 @@ func CreateNodePoolManager(cloudConfigPath string, discoveryOpts cloudprovider.N
 	var err error
 	var configProvider common.ConfigurationProvider
 
-	if os.Getenv(ipconsts.OciUseInstancePrincipalEnvVar) == "true" || os.Getenv(npconsts.OkeUseInstancePrincipalEnvVar) == "true" {
+	if os.Getenv(ipconsts.OciUseWorkloadIdentityEnvVar) == "true" || os.Getenv(npconsts.OciUseWorkloadIdentityEnvVar) == "true" {
```

Comment:
If it's a shared env var, I think we should just have the constant a single time (either in ipconsts or in npconsts). And have a commented section denoting that the below variables are shared.

The only reason the ipconsts and npconsts has an instance principal one is because they actually have different strings and I wanted to keep backwards compatibility.

Reply:
OK, I removed the duplicate and used the instance pool version for both